### ISSUE TITLE
Incorrect YOLO Bounding Box Input Format

### ISSUE THREAD

**Issue by f4str** [Trusted-AI/adversarial-robustness-toolbox#2088]
**Describe the bug**
The YOLOv3 and YOLOv5 estimators require bounding boxes to be in the normalized xywh format as indicated the [YOLOv3 docs](https://github.com/eriklindernoren/PyTorch-YOLOv3/tree/master#train-on-custom-dataset) and [YOLOv5 docs](https://docs.ultralytics.com/yolov5/train_custom_data/#12-create-labels_1). The current `PyTorchYolo` estimator uses the `translate_labels_art_to_yolov3` which does not perform the normalization. Therefore, the inputs for the `compute_loss` and `fit` methods are invalid.

**To Reproduce**
The following sample code can reproduce the issue:

```python
import torch
from art.estimators.object_detection.pytorch_yolo import translate_labels_art_to_yolov3

y = [{
    'boxes': torch.tensor([
        [144., 185., 952., 709.],
        [676., 44., 1150., 691.],
        [429., 433., 517., 704.],
        [979., 316., 1022., 419.],
    ]),
    'labels': torch.tensor([0, 0, 27, 27])
}]
```

If running the `translate_labels_art_to_yolov3`, the output is:

```python
translate_labels_art_to_yolov3(y)
```

The output is:

```txt
tensor([[  0.,   0., 144., 185., 808., 524.],
        [  0.,   0., 676.,  44., 474., 647.],
        [  0.,  27., 429., 433.,  88., 271.],
        [  0.,  27., 979., 316.,  43., 103.]])
```

This is not normalized, the correct format should be normalized like this:

```txt
tensor([[ 0.0000,  0.0000,  0.6850,  0.5588,  1.0100,  0.6550],
        [ 0.0000,  0.0000,  1.1413,  0.4594,  0.5925,  0.8087],
        [ 0.0000, 27.0000,  0.5913,  0.7106,  0.1100,  0.3388],
        [ 0.0000, 27.0000,  1.2506,  0.4594,  0.0538,  0.1287])
```

**Expected behavior**
The `translate_labels_art_to_yolov3` should be fixed to normalize the bounding boxes and produce the correct inputs in the normalized xywh format as indicated by the docs.

**Screenshots**
N/A

**System information (please complete the following information):**
 - OS
 - Python version
 - ART version or commit number
 - TensorFlow / Keras / PyTorch / MXNet version
